### ISSUE TITLE
English language model throws error. Is there a dependency on a specific version of spacy?

### ISSUE THREAD

**Issue by nikhilkmr300** [goru001/inltk#71]
The `predict_next_words` function throws the following error with English:

```
TypeError: blank() got an unexpected keyword argument 'disable'
```

I also tried using the model directly by running `predict` on the pretrained `LanguageLearner` that iNLTK downloads at `en/export.pkl`, but that throws the same error. Looking at the spacy docs, `spacy.blank` currently does not have a parameter called `disable`.

The Hindi language model works fine. I'm guessing the error with English is because, as the docs mention, the model has been taken directly from fastai which probably uses spacy somewhere.

Would appreciate any help. Thanks!

**Comment by chiragjn** [comment]
You are correct, spacy 3.x removed `disable` argument on `spacy.blank`. Downgrading to spacy 2.x should fix it
